<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraOutputBusinessActionWidget</class>
 <widget class="QWidget" name="CameraOutputBusinessActionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>38</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,0">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Output ID:</string>
     </property>
     <property name="buddy">
      <cstring>relayComboBox</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="relayComboBox"/>
   </item>
   <item>
    <widget class="QWidget" name="autoResetWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QCheckBox" name="autoResetCheckBox">
        <property name="text">
         <string>Auto-reset in (sec):</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="autoResetSpinBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="correctionMode">
         <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>60000</number>
        </property>
        <property name="value">
         <number>5</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>relayComboBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
